windowPreview.js

```diff
@@ -36,10 +36,12 @@ var WindowPreviewMenu = class DashToDock_WindowPreviewMenu extends PopupMenu.Pop
         this._source = source;
         this._app = this._source.app;
         let monitorIndex = this._source.monitorIndex;
-
-        this.actor.add_style_class_name('app-menu');
-        this.actor.set_style('max-width: '  + (Main.layoutManager.monitors[monitorIndex].width  - 22) + 'px; ' +
-                             'max-height: ' + (Main.layoutManager.monitors[monitorIndex].height - 22) + 'px;');
+        let scale = St.ThemeContext.get_for_stage(global.stage).scale_factor;
+    
+        this.actor.add_style_class_name('app-well-menu');
```

Comment:
What is the difference between `app-menu` on line 40 with `app-well-menu`

Reply:
I'm not actually sure, but for some reason that was changed up stream. Good catch! I went ahead and updated your commit with that change